src/get-started/modules/ROOT/pages/advanced-configuration/apply.adoc

```diff
@@ -1,26 +1,18 @@
-= Apply a complete applicative policy
+= Apply a security policy
 
-Now we have defined our technique, we will apply it.
-To do so, got to the directives page (*Configuration management -> Directives*)
-and find our new technique (for example by using the filter with "Website"),
-and create a directive.
 
-We will configure the site using the node property we defined in xref:node-management:data.adoc#_classify_nodes_using_node_properties[the node management section].
 
-The syntax to use to access them is `${node.properties[property_name]}`.
-Here we have hierarchical JSON, so we access a sub-key with `${node.properties[property_name][sub_key]}`.
+== Instanciate the policy in a directive
 
-This time our technique takes parameters, so we have to provide them:
+Now we created our technique, we will need to instanciate it and add it to our rule:
 
-* `${node.properties[website][webroot]}`
-* `${node.properties[website][domain]}`
-* `${node.properties[website][port]}`
+. go the *Directives* menu, scroll down, and create the directive *Disable weak ciphers*
```

Comment:
shouldn't you apply it in audit mode, or at least warn that it will change the ssh configuration ?